docs/BR.md

```diff
@@ -1534,9 +1540,36 @@ Additionally, the CA's security program MUST include an annual Risk Assessment t
 
 ### 5.5.1 Types of records archived
 
+The CA and each Delegated Third Party SHALL archive records related to the security of Certificate Systems, Certificate Management Systems, Root CA Systems and Delegated Third Party Systems, especially archive records related to the verification, issuance and revocation of certificate requests and Certificates.
+
+Following is a list of illustrative examples of records archived that are related to the security of Certificate Systems, Certificate Management Systems, Root CA Systems and Delegated Third Party Systems:
+
+- Records of the onboarding process of CA hardware components;
+- Records of change management events for CA software, CA profiles
+- CA ceremony scripts
+- ...
+- ...
+
+The CA SHALL archive records relating to:
+
+1. CA certificate and key lifecycle management event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (1))
+2. Subscriber Certificate lifecycle management event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (2))
+3. Security event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (3))
+
+
 ### 5.5.2 Retention period for archive
 
-The CA SHALL retain all documentation relating to certificate requests and the verification thereof, and all Certificates and revocation thereof, for at least seven years after any Certificate based on that documentation ceases to be valid.
+The CA and each Delegated Third Party SHALL retain, for at least two (2) years:
+
+1. CA certificate and key lifecycle management event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (1)) after the later occurrence of:
+    1. the destruction of the CA Private Key; or
+    2. the revocation or expiration of the final CA Certificate in that set of Certificates that have an X.509v3 `basicConstraints` extension with the `cA` field set to true and which share a common Public Key corresponding to the CA Private Key;
+2. Subscriber Certificate lifecycle management event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (2)) after the expiration of the Subscriber Certificate;
+3. Any security event records (as set forth in [Section 5.4.1](#541-types-of-events-recorded) (3)) after the event occurred; 
+4. All archive records and documentation relating to the verification, issuance, and revocation of certificate requests and Certificates; and
```

Comment:
Should this be "All archived records...." for consistency with bullet #5?

Reply:
fixed